samples/dymaptic.GeoBlazor.Core.Sample.Wasm/dymaptic.GeoBlazor.Core.Sample.Wasm.csproj

```diff
@@ -1,21 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
 
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
the core.sample.shared project is multi-targeted, does this need to be too?

Reply:
Since this is an .exe, multi-targeting isn't really necessary unless we want to run it in multiple runtimes. But that might be useful for testing, so I'll try it out.